src/main/java/com/box/sdk/BoxEvent.java

```diff
@@ -746,5 +746,9 @@ void parseJsonMember(JsonObject.Member member) {
          */
         USER_AUTHENTICATE_OAUTH2_ACCESS_TOKEN_CREATE;
```

Comment:
@ReuvenDamir please change the semi colon to a comma, thank you!

Reply:
@carycheng done. run the test locally and experienced failures in the integration Tests